README.md

```diff
@@ -3342,18 +3346,44 @@ var winstonAdapter = {
     }),
 
     levels: [
+        'off',
         'error',
         'warn',
         'info',
         'debug',
         'silly'
     ],
 
-    log: function (level, className, message, furtherInfo) {
-        this.logger.log(this.levels[level], className + ' ' + message);
+    off: function (level, objectName, message, furtherInfo) {
+        this.logger.log(LogLevel.OFF, objectName, message, furtherInfo);
```

Comment:
like the comment above, `off` is not a required method for the `ILogger interface`